daemonset-templates/nuage-master-config-daemonset.yaml

```diff
@@ -58,9 +58,10 @@ data:
   # on master nodes
   net_yaml_config: |
       networkConfig:
-        clusterNetworkCIDR: 70.70.0.0/16
+        clusterNetworks:
+          - cidr: 70.70.0.0/16
+            hostSubnetLength: 8
```

Comment:
What does the hostSubnetLength signify, perhaps we need to specify that in a comment?